src/ui/public/state_management/__tests__/state.js

```diff
@@ -13,7 +13,7 @@ import StubBrowserStorage from 'test_utils/stub_browser_storage';
 import EventsProvider from 'ui/events';
 
 describe('State Management', function () {
-  const notify = new Notifier();
+  const notifier = new Notifier();
```

Comment:
I agree, only used notify to be consistent with the rest of the code base, but I'm fine changing it.